.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ commands:
       - run:
           name : Configure and run tests
           command: |
-            pip install -r test_requirements.txt
+            pip3 install -r test_requirements.txt
             pytest --junitxml=test_results/result.xml
             echo "Done"
```

Comment:
Can you try setting the environment variable `PYTHONUNBUFFERED=1` and reverting the change to start up YCSB?

See https://docs.python.org/3/using/cmdline.html#envvar-PYTHONUNBUFFERED for more info.